canonical-web-and-design/snapcraft.io
# GitHub webhook - cancel 'in progress' builds

## Done

Cancel ongoing builds when a new build is requested.

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Trigger a snap build twice you should see the old builds cancelled
